src/main/kotlin/study/step3/RacingGame.kt

```diff
@@ -0,0 +1,31 @@
+package study.step3
+
+import kotlin.random.Random
+
+class RacingGame {
+
+    private lateinit var _cars: Array<Car>
+    val cars: Array<Car> get() = _cars
```

Comment:
> 레이싱 게임 클래스에서 자동차 리스트를 멤버로 들고 있는게 맞는 거 같은데 이부분에 대한 테스트 코드를 작성하는게 어려웠어요

이 변수는 테스트 코드에서만 쓰이고 있네요.
테스트 코드를 위해서 내부 정보를 private에서 public으로 공개하는 것보다는 간접적으로 테스트해주는 것이 좋습니다.

Reply:
위 이야기와는 별도로 Array, List와 같이 무언가를 담는 객체는 `val`로 선언해도 외부에서 수정하는 것이 가능합니다.
따라서 외부에 공개해주실 때는 방어적 복사를 하고 넘겨주시는 것이 좋습니다.
```suggestion
    val cars: Array<Car> get() = _cars.copyOf()
```